dashboard-visualiser-superset/importer/config/supersetConfig.js

```diff
@@ -47,39 +49,82 @@
   }
 };
 
+const importZipConfig = async (accessToken) => {
+  const data = new FormData();
+
+  if (CONFIG_FILE) {
+    data.append(
+      'bundle',
+      fs.createReadStream(path.resolve(__dirname, CONFIG_FILE))
+    );
+
+    var config = {
+      method: 'POST',
+      url: `${protocol}://${SUPERSET_SERVICE_NAME}:${SUPERSET_API_PORT}${SUPERSET_IMPORT_PATH}`,
+      headers: {
+        'Content-Type': 'application/zip',
+        Authorization: `Bearer ${accessToken}`,
+        ...data.getHeaders(),
+      },
+      data: data,
+    };
+
+    const res = await axios(config);
+
+    console.log('\n', res.data);
+    console.log('\nConfig imported successfully. exit.');
+  } else {
+    throw new Error(
+      '\nNo path was provided. Please provide the path of the config.'
+    );
+  }
+}
```

Comment:
The importer would crash on these instances and you'd just redeploy the package since the importer would only run on an init and therefore should be a fresh instance. This means it would be safe to remove the package and try again.